packages/devtools_app/lib/src/memory/memory_screen.dart

```diff
@@ -1217,9 +1223,9 @@ class MemoryBodyState extends State<MemoryBody>
                 padding: const EdgeInsets.fromLTRB(5, 0, 0, 4),
                 child: Text('Memory Legend', style: legendHeading),
               ),
-              legendRow(name1: 'Capacity', image1: capacityLegend),
-              legendRow(name1: 'Used', image1: usedLegend),
-              legendRow(name1: 'External', image1: externalLegend),
+              legendRow(name1: 'Allocated', image1: capacityLegend),
```

Comment:
All of these names should be pulled from one const instead of two of the same strings.